alica-supplementary/alica_ros_turtlesim/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
please update main repo Readme links to point to the turtlesim demo and alica_designer_runtime folders in this repo.